src/argowrapper/workflows/argo_workflows/gwas.py

```diff
@@ -193,17 +193,21 @@ def interpret_gwas_workflow_error(step_name: str, step_log: str) -> str:
         """A static method to interpret the error message in the main-log file
         of Failed Retry node
         """
-        if step_name == "run-null-model" and "solve.default" in step_log:
-            show_error = "Small cohort size or unbalanced cohort sizes. A cohort size of about 700 or below may fail for continuous outcome workflow."
-        elif step_name == "run-plots" and "mutate" in step_log:
-            show_error = "Small cohort size or unbalanced cohort sizes."
+        if step_name == "run-null-model" and "system is exactly singular" in step_log:
+            show_error = "The error occurred due to small cohort size or unbalanced cohort sizes. Please ensure that the cohorts selected for your analysis are sufficiently large and balanced."
+        # elif step_name == "run-plots" and "mutate" in step_log:
+        #     show_error = "The error occurred due to small cohort size or unbalanced cohort sizes. Please ensure that the cohorts selected for your analysis are sufficiently large and balanced."
```

Comment:
please remove these and other commented out lines.

Reply:
commented out lines were removed.